src/platform/packages/shared/kbn-esql-ast/src/definitions/utils/autocomplete/helpers.ts

```diff
@@ -309,6 +309,7 @@ export const columnExists = (col: string, context?: ICommandContext) =>
 
 export function getControlSuggestion(
   type: ESQLVariableType,
+  source: ControlTriggerSource = 'smart_suggestion',
```

Comment:
Better to be an enum, you use it in many locations. It is better to handle if this is an enum